src/delegation/DelegationDecoder.ts

```diff
@@ -69,24 +85,40 @@ function verifyRoot(rootId: string): boolean {
   )
 }
 
+export type DelegationNodeRecord = Pick<
```

Comment:
I guess you have already noticed that this causes multiple problems currently. Nevertheless, I am a fan of picking 👍

Reply:
I solved this